### DISCUSSION TITLE
show lines for the routes that everyone is taking on the /statuses/active map

### DISCUSSION THREAD

**Discussion by thibaultmol** [Traewelling/traewelling#2549] [Ideas / Feature Requests]
Currently you can only see circles with people's pfp's. But it would nice if there was a toggle to also show all the lines of the routes people are taking. Because someone might be near you but they're moving away from you instead.

**Comment by HerrLevin** [comment]
We had this in the past but there were so many check-ins that the browsers crashed. That's why we added circles instead. Sorry. 😕

**Reply by thibaultmol** [reply]
Maybe switching the map to maplibre gl js would allow for it? 
but I get that's a lot of work to do.
https://github.com/maplibre/maplibre-gl-js

**Reply by HerrLevin** [reply]
It might. I've also considered [d3](https://d3js.org/) but ultimately decided against it because I wanted to keep the frustration in my free time fairly low. 😅

We might revisit this topic once we've reduced the other problem zones around here.

I will open this discussion again. If anyone wants to implement this (even if it's just a poc), they are more than welcome to do so. :)
